raiden/raiden_event_handler.py

```diff
@@ -540,3 +540,24 @@ def handle_contract_send_channelsettle(
             partner_locksroot=partner_locksroot,
             block_identifier='latest',
         )
+
+
+class RaidenMonitoringEventHandler(RaidenEventHandler):
+    # pylint: disable=no-self-use
+
+    def on_raiden_event(self, raiden: RaidenService, event: Event):
+        if type(event) == EventPaymentReceivedSuccess:
+            self.handle_received_payment(raiden, event)
+
+        # Call base class to trigger other side-effects
```

Comment:
nitpick: I would phrase it as: "Call base class for processing of all other event types"

Reply:
It might be the same event again as well, this is just a decorator.

Reply:
Hmm indeed. Disregard my comment. Was thinking of something completely different.